.travis.yml

```diff
@@ -3,10 +3,18 @@ os:
   - linux
   - osx
 julia:
-  - 0.5
-  - 0.6
+  - 1.0
+  - 1.1
+  - 1.2
```

Comment:
Since 1.3 is the current releast, and 1.0 is the long-term stable, I don't think we need to test on 1.1 and 1.2. I'm fine to leave them, just makes CI take longer.